packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -597,6 +597,22 @@ export default class Meeting extends StatelessWebexPlugin {
     this.setUpLocusMediaSharesListener();
     this.setUpLocusInfoMeetingInfoListener();
     this.setUpLocusInfoAssignHostListener();
+    this.mediaInactiveHandler();
+  }
+
+
+  /**
+   * Set up the locus info listener for meetings disconnected due to inactivity
+   * @returns {undefined}
+   * @private
+   * @memberof Meeting
+   */
+  mediaInactiveHandler() {
+    this.locusInfo.on(EVENTS.DISCONNECT_DUE_TO_INACTIVITY, (res) => {
+      // TODO: need to send metric for locus disconnect
+      LoggerProxy.logger.error(`Meeting:index#mediaInactiveHandler --> Meeting disconnected due to ${res.reason}`);
```

Comment:
Don't we know that this is due to inactivity? What does the reason give us?

```suggestion
      LoggerProxy.logger.error(`Meeting:index#mediaInactiveHandler --> Meeting disconnected due to inactivity: ${res.reason}`);
```